mava/systems/jax/mappo/components.py

```diff
@@ -55,15 +55,23 @@ def __init__(
         self,
         config: ExtrasLogProbSpecConfig = ExtrasLogProbSpecConfig(),
     ):
-        """_summary_
+        """Class that adds log probs to the extras spec
 
         Args:
-            config : _description_.
+            config : configuration for current algorithm
         """
         self.config = config
 
     def on_building_init_end(self, builder: SystemBuilder) -> None:
-        """[summary]"""
+        """Create extra specs after builder has been initialised
+
+        Args:
+            builder: builds system
```

Comment:
same here: in the components docs PR we have builder: SystemBuilder. I agree that descriptions are needed when the var name is not intuitive/ it is not obvious what the arg does

Reply:
Changed to be more in line with the other docs.